card/src/main/java/com/adyen/checkout/card/util/AddressFormUtils.kt

```diff
@@ -57,13 +58,31 @@ internal object AddressFormUtils {
                     countryList
                 }
 
-                val defaultCountryCode = addressConfiguration.defaultCountryCode.orEmpty()
+                val defaultCountryCode = getInitialCountryCode(
+                    shopperLocale = shopperLocale,
+                    addressConfiguration = addressConfiguration
+                )
                 markAddressListItemSelected(mapToListItem(filteredCountryList), defaultCountryCode)
             }
             else -> emptyList()
         }
     }
 
+    /**
+     * get initial country code.
+     *
+     * @param shopperLocale Locale value from dropIn configuration or card component configuration.
+     * @param addressConfiguration
```

Comment:
Optional: On second thought we could do something like: `@param addressConfiguration Full Address Configuration`

Reply:
it's self explanatory we don't need to add that.

Reply:
also it's limited to `AddressConfiguration.FullAddress` type